salt/modules/virt.py

```diff
@@ -1358,7 +1391,7 @@ def init(name,
     .. rubric:: Network Interfaces Definitions
 
     Network interfaces dictionaries can contain the following properties:
-
+boot
```

Comment:
Wrong place for this documentation item

Reply:
I didn't know when did this line get inserted. I will go git diff and check carefully every time before I commit

Reply:
happens :) but you'll need to add some doc for the new boot properties that you are adding